app/src/main/assets/applozic-settings.json

```diff
@@ -154,5 +154,6 @@
     "submit": false,
     "link": false,
     "quickReply": false
-  }
+  },
+  "receiverNameTextColor": "#5C6677"
```

Comment:
Is it receiver or sender? Name seems confusing

Reply:
Its receiver. It can be agent or Bot for User.

Reply:
If you send me a message then I m the receiver?

Reply:
Left side one will be the sender right?

Reply:
Yes. But I think it is easier as receiver as the word is being used in several other settings like receivedMessageTextColor, etc. Maybe we can make it as receivedMessageTitleColor.

Reply:
receivedMessageTitleColor is better if we have something for message.
This is for name so we can go ahead with existing used